### ISSUE TITLE
Non-descriptive confirmation message on deleting a Category

### ISSUE THREAD

**Issue by tuxpiper** [ushahidi/platform#4732]
https://github.com/ushahidi/platform/assets/8624777/043171b5-374c-47a3-b2c0-a2ce59d8413e

A toast comes up with the message "bulk destroy_success_countless" on deleting a category from the Bulk actions way. Let's have a proper descriptive plain english message instead.

---
Created automatically from USH-889 (https://linear.app/ushahidi/issue/USH-889/non-descriptive-confirmation-message-on-deleting-a-category)

**Comment by Angamanga** [comment]
@deveshidwivedi Welcome to Ushahidi! In order to start an issue, you need to follow the steps in this discussion: https://github.com/ushahidi/platform/discussions/4713 Have you set up your developer environment yet?

**Comment by MundiaNderi** [comment]
Hey @Angamanga I've set up my developer environment and I would like to be assigned this issue.

![Image](https://github.com/ushahidi/platform/assets/98798719/8a24f521-8aa5-490c-9d70-fec86986b932)

**Comment by Remi-dee** [comment]
Hello @Angamanga, i am an Outreachy applicant and i've got my developer environment up and running. I'd like to work on this issue

**Comment by RWUBAKWANAYO** [comment]
Hi @Angamanga , as an Outreachy applicant, I've completed the setup of the project locally. I'd like to work on this issue, could you please assign it to me?



![Image](https://github.com/ushahidi/platform/assets/68381641/c01ddfc6-d51a-48ea-8158-f33d425334ae)
![Image](https://github.com/ushahidi/platform/assets/68381641/6ee3e6ef-3779-4baf-9b83-44d7f5ea616a)

**Comment by felabel** [comment]
Hi @Angamanga I have completed my setup as shown below and would like to take up this task;
I have also spent some time digging around the category component and noticed few other issues;
1. There is no error  message if a category isn't successfully created
2. There is no success message when a category is successfully created.

I believe this experience can be improved for the users.
I have tried to show a generic error message while playing around with the code below. I would be happy to take this up, thank you.

![Image](https://github.com/ushahidi/platform/assets/63490084/122d63e1-6e94-4569-b0c0-f26ea85fa737)




![Image](https://github.com/ushahidi/platform/assets/63490084/6c32830d-61a6-493f-b52f-dcca57dc3bf6)

**Comment by amooesther** [comment]
Hello @tuxpiper @Angamanga, i am an outreachy applicant, I have setup my developer environment and will like to work on this project

![Image](https://github.com/ushahidi/platform/assets/131756511/82fb7677-3d5d-4f7f-9e57-6e8d28992efd)

**Comment by Angamanga** [comment]
@felabel Thank you for raising new issues! Please add them here: https://github.com/ushahidi/platform/issues/new/choose so we can triage and look at them.

The rest of you, since this was already worked on, I have added 2 issues that are open for everyone to start working on straight away, these are https://github.com/ushahidi/platform/issues/4749 and https://github.com/ushahidi/platform/issues/4750. Thank you 🙏

**Comment by felabel** [comment]
I had raised a PR for the error handling - https://github.com/ushahidi/platform-client-mzima/pull/823
Please take a look

**Comment by felabel** [comment]
@Angamanga I used tor link you dropped in your comment to create an issue and selected the bug category but the issue wasn’t still labeled. Please what am I missing ?

**Comment by Angamanga** [comment]
This is done
